package.json

```diff
@@ -48,6 +48,7 @@
     "azure-asm-subscription": "0.10.2",
     "azure-asm-trafficmanager": "0.10.4",
     "azure-asm-website": "0.10.5",
+    "azure-arm-analysisservices": "1.0.0",
```

Comment:
this is not required to be in the PR. This will cause the PR to fail. We will add this when we publish the new rollup version.